src/anyon/anyon.jl

```diff
@@ -238,47 +276,72 @@ julia> get_qubits_distance(3, 24, connectivity)
 ```
 
 """
-get_qubits_distance(target_1::Int, target_2::Int, ::LineConnectivity)::Int =
+function get_qubits_distance(target_1::Int, target_2::Int, c::LineConnectivity)::Real
+    for e in c.excluded_positions
+        if target_1 ≤ e ≤ target_2
+            return NaN
```

Comment:
I think we should use Infinity to indicate a disconnect.

Reply:
updated